### DISCUSSION TITLE
Sometimes the application does not start when called with the "--connect ID" parameter

### DISCUSSION THREAD

**Discussion by ghost** [rustdesk/rustdesk#6272] [Q&A] [answered]
### Bug Description

It is difficult to reproduce this behavior. I tried to record a video but the video was too long, as the problem does not always occur immediately. When I call RustDesk using the "--connect ID" parameter, sometimes it does not open the application connection window (the one that asks for the password). It's something intermittent, but I'm sure it didn't happen before. Some time ago I experienced this same problem, as I mistakenly installed a newer version of Flutter than the one used in the project until then, version 3.10.6. Coincidence or not, I noticed that the Flutter version of the project was recently updated to version 3.13.9. When the problem occurs, monitoring the processes through the task manager I noticed that the "rustdesk.exe" process is created and terminated shortly afterwards (it is something very fast and difficult to notice). On some occasions, after several call attempts, several "zombie" rustdesk.exe processes remain running, as can be seen in the image I posted here. For those who access RustDesk using the normal method, clicking on the application icon and then entering the ID in the appropriate field, this is not a problem. However, I integrate RustDesk with another system, which directly calls RustDesk using the "--connect ID" parameter. In my case, this is quite disruptive, as it is necessary to make several calls until the application opens.

### How to Reproduce

Create a shortcut for RustDesk.exe and enter the "--connect ID" parameter... Make several attempts by clicking on the shortcut. Most of the time the application opens normally and the window asking for the password appears. However, on some occasions the application screen simply is not displayed.

### Expected Behavior

Most of the time the application opens normally and the window asking for the password appears. However, on some occasions the application screen simply is not displayed.

### Operating system(s) on local side and remote side

Windows 11

### RustDesk Version(s) on local side and remote side

1.2.4 (Nightly) 

### Screenshots

<img width="621" alt="tela" src="https://github.com/rustdesk/rustdesk/assets/129885668/91014cc0-e93c-4590-96da-0b3826ec0d5c">

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/issues/6273, my mistake

**Comment by rustdesk** [comment, marked as answer]
https://github.com/rustdesk/rustdesk/issues/6273#issuecomment-1798233520
